gem/lib/pupperware/spec_helper.rb

```diff
@@ -217,6 +230,15 @@ def wait_on_puppetdb_status(seconds = 240)
     end
   end
 
+  ######################################################################
+  # PE Console Services Helper
+  ######################################################################
+  def unrevoke_console_admin_user(postgres_container_name="postgres")
+    query = "docker-compose --no-ansi exec -T #{postgres_container_name} psql --username=puppetdb --dbname=pe-rbac --command \"UPDATE subjects SET is_revoked = 'f' WHERE login='admin';\""
```

Comment:
@cthorn42 I think this should use the new `docker_compose` method so it properly uses the overrides files (and the other flags that we want all calls to `docker-compose` to have, like `--no-ansi`).

This PR should change all existing calls to `run_command("docker-compose ...")` to `docker_compose("...")`

Reply:
Looks like I missed some spots in the original update too...

I'll update 'em all